src/app/components/team/AssignedToMe.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { QueryRenderer, graphql } from 'react-relay/compat';
+import Relay from 'react-relay/classic';
+import { FormattedMessage } from 'react-intl';
+import ErrorBoundary from '../error/ErrorBoundary';
+import Search from '../search/Search';
+import PersonIcon from '../../icons/person.svg';
+
+const defaultFilters = {
+  assigned_to: ['me'],
```

Comment:
@caiosba this is an idea, the api could recognize this constant and we could skip the query to get current user dbid, and the exported defaultQuery would also work

Reply:
Yes, good point. Let's create a ticket for that after we have the architecture discussion.